compare/compare.py

```diff
@@ -108,15 +128,26 @@ def __init__(self, content, source1, source2):
         self.content = content
         self.source1 = source1
         self.source2 = source2
+
+        distro_url = source1.distro_url
+        distro_view = source1.distro_view
```

Comment:
This needs to be source2 for both of these.  Otherwise the numbers are flipped.  We usually do
./compare.py -c fedora rhel